### DISCUSSION TITLE
M4b audiobooks

### DISCUSSION THREAD

**Discussion by Silther** [zyrouge/symphony#397] [Q&A]
Does the app support m4b audiobooks?

**Comment by zyrouge** [comment]
`.m4b` audio files are supported. If audiobooks are similar, it should work.

**Comment by Silther** [comment]
Are chapters also supported?
Like showing one m4b file as mutliple chapters in the album view.

**Reply by zyrouge** [reply]
I'm pretty sure that's not supported.

**Comment by zyrouge** [comment]
Closing this as "beyond scope". Feel free to let out your thoughts.

**Reply by Silther** [reply]
makes sense, most people wont use non mp3 audiobooks anyway and it probably would be pretty tedious to implement.
